content/index.js

```diff
@@ -107,6 +107,15 @@ function($scope, $rootScope, $routeParams, $location, $uibModal, loginStatusProv
 		return viewLocation === pageService.GetCurrentPage();
 	};
 
+	$scope.isPublicProfile = function(viewLocation) {
+		if(pageService.GetCurrentPage().indexOf('/view') >= 0){
```

Comment:
The if block is redundant, since this would be the same as:
`return pageService.GetCurrentPage().indexOf('/view') >= 0;`
Right?

and rather than `indexOf` there is `includes` which can simplify this even further.  so
`return pageService.GetCurrentPage().includes('/view');`